packages/ethereum-contracts/contracts/utils/ACL.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract ACL is AccessControl {
```

Comment:
Please always try to give it a distinctive name, it doesn't deserve such a undecorated name since it has specific implementation of how admin role are assigned and transferred. 

Any thing with an ajective is better: "TransferrableAdminACL", "SimpleAdmin", etc.